packages/next-swc/crates/next-dev/benches/bundler.rs

```diff
@@ -44,17 +46,27 @@ impl Bundler for TurboNext {
     }
 
     fn prepare(&self, install_dir: &Path) -> Result<()> {
-        let package_json = include_str!("../../../../next/package.json");
-        let data: serde_json::Value = serde_json::from_str(package_json)?;
-        let version = data
-            .as_object()
-            .unwrap()
-            .get("version")
-            .unwrap()
-            .as_str()
-            .unwrap();
-        npm::install(install_dir, &[NpmPackage::new("next", version)])
-            .context("failed to install from npm")?;
+        let repo_path = current_dir()?.join("../../../..").canonicalize()?;
```

Comment:
This means it has to be executed in the crate's directory?
isn't it more likely to execute it from the root of the repo where the `Cargo.toml` is?

Reply:
For some reason, this gives me `next.js/packages/next-swc/crates/next-dev` when running it with `cargo test --benches -p next-dev --release --no-fail-fast "bench_hydration/TurboNextDev SSR"`. I'll change this to use `CARGO_MANIFEST_DIR` like our other tests.